microsoft/SqlNexus
# Multiple SQL Nexus fixes (#250, #251, #238, #254)

1. Allow the ability to minimize Cmd prompt window when Relog.exe imports Performance .BLG files into SQL Nexus

   ![image](https://github.com/microsoft/SqlNexus/assets/23519517/7dd81357-c5de-4c1b-b40a-3ed21f3e19a3)

1. Allow the ability to import XEL files using local server time, rather than UTC time (default) via ReadTrace
 
   ![image](https://github.com/microsoft/SqlNexus/assets/23519517/0d8a9d26-a862-4f61-afe7-4a916756b593)

1. Fixed "Save my options" functionality in the import window to ensure the import options were really saved. Some of the options weren't saved before even though the GUI shows the boxes are checked. 
  
   ![image](https://github.com/microsoft/SqlNexus/assets/23519517/3d330922-5e50-45b0-be98-41c3160d3c32)

    To test this last fix, 
     - select "Save my options" 
     - then enable one of the import options (for example minimize Cmd Window (relog.exe) ) 
     - Do the import. 
     - Then close the Import dialog 
     - Open it back up 
     - Re-import and ensure the option is still working
 
1. REDO_THREAD_PENDING_WORK wait type is now ignorable type, which means it will not show up in the Bottleneck Analysis report as a bar in the bar chart.
